pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -329,6 +350,26 @@ func newRunnerPodForCR(cr *litmuschaosv1alpha1.ChaosEngine, aUUID types.UID, aEx
 				WithArgumentsNew([]string{"-c", "ansible-playbook ./executor/test.yml -i /etc/ansible/hosts; exit 0"}).
 				WithEnvsNew(getChaosRunnerENV(cr, aExList)),
 		).
+		Build()
+	if err != nil {
+		return nil, err
+	}
+	return podObj, nil
```

Comment:
Change this `podObj` variable to `runnerPod`

Reply:
Done, pushing commit